clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -476,6 +476,16 @@ protected void handleUserCommand() {
 
     Command.setAuthenticationMode(auth, userName);
 
+    if (CommandActions.LIST.equals(command)) {
+      newListUsers(url, ignore, metalake).handle();
+      return;
+    }
+
+    if (Objects.isNull(user)) {
```

Comment:
Why not use user == null?

Reply:
fix, thank you Justin, plz review it again.